### DISCUSSION TITLE
Do I need auth on the proxy or can I use open proxies?

### DISCUSSION THREAD

**Discussion by 6c6-dev-100** [KevinBytesTheDust/CVAmp#188] [General]
Is authentication required for the proxy, or is it acceptable to utilize open proxies?

**Comment by jlplenio** [comment]
Hello @drmabuse19. You can use proxies without auth or open proxies. You have to follow the following format in your proxy_list.txt file:
https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Proxies#configure-proxies

But it is very likely, that you will encounter many problems, see here: 
https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Proxies#public-proxies
